docs/stages.adoc

```diff
@@ -36,9 +36,10 @@ I fill out the registration form
 ----
 The nested steps will appear as a collapsible group in the HTML report.
 
+[#_overriding_the_default_reporting]
 === Overriding the Default Reporting
 
-Sometimes it is necessary to override the default way of the step reporting. For example, if you want to use special characters that are not allowed in Java methods names, such as `(`, `+`, or `%`. You can then use the `@As` annotation to specify what JGiven should put into the report.
+Sometimes it is necessary to override the default way of the step reporting.For example, if you want to use special characters that are not allowed in Java methods names, such as `(`, `+`, or `%`.You can then use the `@As` annotation to specify what JGiven should put into the report.
```

Comment:
... reporting. [space] For example ....
... such as (, + or % [comma], [you] can ...